test/README.md

```diff
@@ -0,0 +1,15 @@
+# Running tests
+
+This folder has all the system test files for lxd.
+
+## Dependencies
+
+You'll need the following dependencies installed before you can run them:
+
+    sudo apt-get install jq sqlite3 golang-go
```

Comment:
Ohh nice, alright, then I agree there doesn't need to be an extra source of (un)truth.

I'll close this PR and send a new one taking another approach on the same thing instead.